specificatie/gezag/gezagsrelatie-v1.yaml

```diff
@@ -33,6 +33,7 @@ components:
           burgerservicenummer: '999998894'
 
     TweehoofdigOuderlijkGezag:
+      description: 'De naam van dit type is onjuist. De juiste term is "gezamenlijk ouderlijk gezag", en moet worden gebruikt in de communicatie met de eindgebruiker. In de eerst volgende major versie zal het type worden aangepast.'
```

Comment:
Hier geldt hetzelfde als hierboven

Reply:
wordt deze alleen gebruikt voor de data-service interface? of ook voor de informatie-service interface? want in de informatie-service interface kan het nog niet worden aangepast, toch?

Reply:
Deze wordt alleen gebruikt voor de data-service interface. De OAS componenten die worden gebruikt in de informatie-service interface zitten onder mapje `brp-api/...` in plaats van `brp/...`.